### PR TITLE
AB module option reordering

### PR DESCRIPTION
**Pull request by ulteq**

### CODE REVIEW

**Comment by jaynus** [comment]
This is going to break peoples server configs and they are going to need to re-export them. 

We need to avoid doing changes like this.

**Comment by ulteq** [comment]
It'll only break if they're not using the defaults? And the worst case is that it is reset back to default?

**Comment by esteldunedain** [comment]
I don't think this won't break at all. Worst case scenario we may need to add code to the settings code to ignore settings on the userconfig that don't exist anymore (although I think it works decently even under those conditions, just that it loads obsolete settings).

File: addons/viewdistance/ACE_Settings.hpp [hunk 1]
```diff
@@ -8,24 +8,24 @@ class ACE_Settings {
     class GVAR(viewDistanceOnFoot) {
         typeName = "SCALAR";
         isClientSettable = 1;
-        value = 0; // index, NOT value // Can set it to client's actual viewdistance in the init function once ACE_Settings supports numbers (if ever).
-        values[] = {"Video settings","500","1000","1500","2000","2500","3000","3500","4000","5000","6000","7000","8000","9000","10000"}; // Values also need to be changed in functions/fnc_returnValue.sqf
+        value = 11; // index, NOT value // Can set it to client's actual viewdistance in the init function once ACE_Settings supports numbers (if ever).
```
Comment (esteldunedain) [inline addons/viewdistance/ACE_Settings.hpp]: No, this should no reverse the changes I made today. Although it's probably github being funny

**Comment by esteldunedain** [comment]
> @glowbal @esteldunedain: Do we have any mechanic to handle renaming of these settings for profileNamespace/serverconfig? 

@jaynus, to clarify my last comment. Obsolete settings from the profile are just not loaded, so renaming doesn't create any problem. The only thing that would break is if we change a setting TYPE without renaming it (e.g., change a number to an array).

**Comment by ulteq** [comment]
So what do we do about it?

**Pull request merged**